engine/api/workflowtemplate/dao_test.go

```diff
@@ -0,0 +1,56 @@
+package workflowtemplate
+
+import (
+	"testing"
+
+	"github.com/ovh/cds/engine/api/test"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTemplate(t *testing.T) {
+	db := &test.SqlExecutorMock{}
```

Comment:
nope we want to test with a real database... the goal is too test the queries

Reply:
answer on comment for TestAggregateAuditsOnWorkflowTemplate

Reply:
done